src/components/input/input.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { Dispatch, InputHTMLAttributes, SetStateAction } from "react";
+
+export interface IInputProps extends InputHTMLAttributes<HTMLInputElement> {
+  label: string;
+}
+export interface IInputContainerProps {
+  children: any;
```

Comment:
🦆 this any can be substituted by `ReactElement` imported from react itself :)

Reply:
🥔 Done! 💪